forms-flow-web/src/components/Modals/ActionModal.js

```diff
@@ -115,8 +117,18 @@ const ActionModal = React.memo(({ newActionModal, onClose, CategoryType, onActio
                                     className=""
                                     dataTestid="export-workflow-button"
                                     ariaLabel="Export Workflow"
+                                    onClick={() => handleAction("EXPORT")}
+                                />
+                           <CustomButton
+                                    variant="secondary"
+                                    size="sm"
+                                    label="Delete"
+                                    icon={<CloseIcon color="#253DF4" />}
```

Comment:
in the entire file , wherever we have added color="#253DF4" , please replace with css variable 
var(--ff-base-600) . It is present in entire page. 
Also if we dont have classNames to add , we can remove it from all the CustomButtons in this page 

@abilpraju-aot